### ISSUE TITLE
Dependencies question

### ISSUE THREAD

**Issue by gilfreund** [bm2-lab/iGWOS#4]
Hello,

In the dependencies, both Python 2.7 and 3.7 are listed. Are two python environments are required?

**Comment by jafferyoung** [comment]
Yes. You can refer to the Dockerfile on the Github, which has built the environment based on ubuntu. Then you can easily run iGWOS in the environment. @gilfreund
